vendor/assets/javascripts/es5-shim.js

```diff
@@ -0,0 +1,1572 @@
+/*!
```

Comment:
We really just need it for the test environment (CircleCI has an older `phantomjs` version without `Function.bind`, and building the latest version on the vm turned out to be non-trivial when we tried).

I think we can pull it in via rails-assets and only source it in the js specs.